### ISSUE TITLE
Add support for laravel 6.0

### ISSUE THREAD

**Issue by seedgabo** [eduardoarandah/backpacklogviewer#1]

**Comment by timbertens** [comment]
Could you add support for 6.0 please?

**Comment by eduardoarandah** [comment]
is there any issues with this running on laravel 6?

**Comment by timbertens** [comment]
yes, see below

- eduardoarandah/backpacklogviewer 1.0.5.1 requires illuminate/support ~5.1 -> satisfiable by laravel/framework[v5.8.35], illuminate/support[5.7.17, 5.7.18, 5.7.19, v5.1.1, v5.1.13, v5.1.16, v5.1.2, v5.1.20, v5.1.22, v5.1.25, v5.1.28, v5.1.30, v5.1.31, v5.1.41, v5.1.6, v5.1.8, v5.2.0, v5.2.19, v5.2.21, v5.2.24, v5.2.25, v5.2.26, v5.2.27, v5.2.28, v5.2.31, v5.2.32, v5.2.37, v5.2.43, v5.2.45, v5.2.6, v5.2.7, v5.3.0, v5.3.16, v5.3.23, v5.3.4, v5.4.0, v5.4.13, v5.4.17, v5.4.19, v5.4.27, v5.4.36, v5.4.9, v5.5.0, v5.5.16, v5.5.17, v5.5.2, v5.5.28, v5.5.33, v5.5.34, v5.5.35, v5.5.36, v5.5.37, v5.5.39, v5.5.40, v5.5.41, v5.5.43, v5.5.44, v5.6.0, v5.6.1, v5.6.10, v5.6.11, v5.6.12, v5.6.13, v5.6.14, v5.6.15, v5.6.16, v5.6.17, v5.6.19, v5.6.2, v5.6.20, v5.6.21, v5.6.22, v5.6.23, v5.6.24, v5.6.25, v5.6.26, v5.6.27, v5.6.28, v5.6.29, v5.6.3, v5.6.30, v5.6.31, v5.6.32, v5.6.33, v5.6.34, v5.6.35, v5.6.36, v5.6.37, v5.6.38, v5.6.39, v5.6.4, v5.6.5, v5.6.6, v5.6.7, v5.6.8, v5.6.9, v5.7.0, v5.7.1, v5.7.10, v5.7.11, v5.7.15, v5.7.2, v5.7.20, v5.7.21, v5.7.22, v5.7.23, v5.7.26, v5.7.27, v5.7.28, v5.7.3, v5.7.4, v5.7.5, v5.7.6, v5.7.7, v5.7.8, v5.7.9, v5.8.0, v5.8.11, v5.8.12, v5.8.14, v5.8.15, v5.8.17, v5.8.18, v5.8.19, v5.8.2, v5.8.20, v5.8.22, v5.8.24, v5.8.27, v5.8.28, v5.8.29, v5.8.3, v5.8.30, v5.8.31, v5.8.32, v5.8.33, v5.8.34, v5.8.35, v5.8.4, v5.8.8, v5.8.9].

**Comment by eduardoarandah** [comment]
@timbertens oh, I see. 

If you don't mind, could you fork it, manually change composer.json
https://github.com/eduardoarandah/backpacklogviewer/blob/master/composer.json#L22

and see if works fine? 

I'd happily accept accept your pull request.

I don't have a laravel project right now to test it

**Comment by juventus18** [comment]
Forgot to mark this fixed in my PR, but it can be closed now.

**Comment by juventus18** [comment]
I was just going to add something to this: can you also mark a new tag/release to make it actually installable? I think it would be good to bump up to version 2.0.0 to get on track with semantic versioning (or even version 6 if you want to go parallel to laravel)

**Comment by eduardoarandah** [comment]
Sure! Forgot to tag it, thanks @juventus18 !

I've read about semantic versioning in laravel and I think it's very nice!

This package is basically an adaptation of Arcanedev/LogViewer for the backpack backend. 

I see minor improvements in the logViewer since 5.0:

https://github.com/ARCANEDEV/LogViewer/releases

and a big release (v4) for backpack since v3: 

https://github.com/Laravel-Backpack/CRUD/releases

So there's the option of going semantic with LogViewer, Backpack or Laravel, what do you think?

**Comment by juventus18** [comment]
I think its pretty arbitrary right now what you pick the version to be, as long as you follow the https://semver.org/ guidelines going forward. I'd probably just go with v2.0.0 since, as you point out, there are a lot of targets to try and make a parallel version, so it doesn't make much sense.

**Comment by justinmoh** [comment]
@eduardoarandah latest release wasn't on `packagist`. Would you push the update please?

**Comment by eduardoarandah** [comment]
@justinmoh you're right! looks like github deprecated the sync method I was using in favor of webhooks. 

It's synced now ;-)
